climbot5d:
  # joint_names
  joint_names: [G0_Joint,I1_Joint, T2_Joint, T3_Joint, T4_Joint, I5_Joint,G6_Joint]
  
  # Publish all joint states -----------------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Joint Position Controllers ---------------------------------------
  joint_position_controller:
    type: position_controllers/JointGroupPositionController
    joints: 
      - I1_Joint
      - T2_Joint
      - T3_Joint
      - T4_Joint
      - I5_Joint
    pid: {p: 100.0, i: 0.01, d: 10.0}


  jointG0_effort_controller:
    type: effort_controllers/JointEffortController 
    joint: G0_Joint
    pid: {p: 100.0, i: 0.01, d: 10.0}


  jointG0_1_effort_controller:
    type: effort_controllers/JointEffortController 
    joint: G0_Joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}

  jointG6_effort_controller:
    type: effort_controllers/JointEffortController 
    joint: G6_Joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  jointG6_1_effort_controller:
    type: effort_controllers/JointEffortController 
    joint: G6_Joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}


